cockatrice/src/logger.cpp

```diff
@@ -51,16 +51,18 @@ void Logger::closeLogfileSession()
 void Logger::log(QtMsgType /* type */, const QMessageLogContext & /* ctx */, const QString &message)
 {
     logBuffer.append(message);
-    if(logBuffer.size() > LOGGER_MAX_ENTRIES)
-        logBuffer.removeFirst();
+    if (logBuffer.size() > LOGGER_MAX_ENTRIES)
+        logBuffer.clear();
```

Comment:
Sorry for the late review, but this doesn't seems correct.
The original idea was to have a circular buffer that contains up to `LOGGER_MAX_ENTRIES` entries.
When the `LOGGER_MAX_ENTRIES + 1` entry gets added, the older one gets discarded.
With this change, when the `LOGGER_MAX_ENTRIES + 1` gets added, everything gets removed from the log, including the message just added.